.gitignore

```diff
@@ -0,0 +1,2 @@
+/.gradle/
```

Comment:
This makes sure files create by a build or running the application which shouldn't be part of the source code don't end up committed to Git